src/Files.App/BaseLayout.cs

```diff
@@ -509,6 +510,9 @@ private void FolderSettings_GroupOptionPreferenceUpdated(object? sender, GroupOp
 		private void FolderSettings_GroupDirectionPreferenceUpdated(object? sender, SortDirection e)
 			=> GroupPreferenceUpdated();
 
+		private void FolderSettings_GroupByDateUnitPreferenceUpdated(object? sender, GroupByDateUnit e)
+			=> GroupPreferenceUpdated();
```

Comment:
I think you should change this to use brackets

Reply:
@0x5bfa says the opposite.
https://github.com/files-community/Files/pull/12288#discussion_r1186664959

I think it is better to be concise.

Reply:
This is going to be my job. So let me be clear.

If a method have only return, we have to make them lambda.

```cs
public bool Tasks()
{
    return true;
}
```

Only this case, I approve.

Reply:
The coding convention in Files is to use brackets in most cases.

Reply:
I changed to using brackets. Also added async-await to remove warnings.